Resources/Prototypes/_ES/GameRules/degradation_events.yml

```diff
@@ -0,0 +1,81 @@
+- type: entity
+  parent: BaseGameRule
+  id: ESBaseDegradationEvent
+  name: Unnamed Degradation Event
+  abstract: true
+  components:
+  - type: ESDegradationEvent
+  - type: StationEvent
+    earliestStart: 0
+    reoccurrenceDelay: 0
+    occursDuringRoundEnd: false
+    weight: 10
+
+- type: entityTable
+  id: ESDegradationEvents
+  table:
+    all:
+    - id: ESDegradationEventAirlocks
+    - id: ESDegradationEventCommunicationsConsole
+    - id: ESDegradationEventCriminalRecords
+    - id: ESDegradationEventTelecoms
+    - id: ESDegradationEventCrewMonitoring
+    - id: ESDegradationEventIDCardComputer
```

Comment:
imo airlocks and telecoms should be weighted slightly higher than the others (since theres more of them), airlocks moreso